Havit.Blazor.Components.Web.Bootstrap/wwwroot/HxScrollToValidationMessageContainer.js

```diff
@@ -0,0 +1,58 @@
+﻿const findGroupFromMessage = (messageElement) => {
```

Comment:
What is the advantage of this approach in opposite to regular (private, not exported) function in the module?